core-plugins/src/test/java/io/cdap/plugin/transform/ProjectionTransformTest.java

```diff
@@ -51,6 +54,14 @@
     .set("stringField", "bar")
     .build();
 
+  private static final String validationExceptionMessage = "Errors were encountered during validation.";
```

Comment:
nit: private static final constant should be all caps with underscores. I think the checkstyle will complain about this.

Reply:
also, exception messages should not be checked in unit tests. There is no contract on messages, they can change at any time.

Reply:
Fixed